### DISCUSSION TITLE
Resizable window

### DISCUSSION THREAD

**Discussion by dougreichard** [raylib-extras/rlImGui#31] [Q&A] [answered]
I'm trying to switch from SDL to raylib/glImGui but resizing isn't working.

What is the proper way to create a resizable window. When the window is resized ImGui seems to draw in correctly.

**Comment by JeffM2501** [comment]
rlImGui uses the current window size to tell ImGui what to do so it respects the raylib window size.

If you pass in `SetConfigFlags(FLAG_WINDOW_RESIZABLE) `before you call `InitWindow `that will make they raylib window resizable. You don't have to do anything with ImGui to support that.

If that's not what you are talking about then I need more info.

**Reply by dougreichard** [reply]
I am doing that exactly, but resizing the windows the gui render gets corrupted. 

The simple example with FLAG_WINDOW_RESIZABLE
![Screenshot 2023-08-31 184103](https://github.com/raylib-extras/rlImGui/assets/3019243/79846d89-edc6-4d33-bd61-62e71e2708bb)

The same app after resizing it
![Screenshot 2023-08-31 184031](https://github.com/raylib-extras/rlImGui/assets/3019243/4b28d298-5678-4fee-a610-803b707a46c3)

**Comment by JeffM2501** [comment]
It's the stencil buffer, it's screwing up. I'll take a look.

**Comment by dougreichard** [comment]
Thanks. For the record it is a 4k display.
The editor seems to work. With the exception that the parrots turn black on
resize.

**Comment by JeffM2501** [comment, marked as answer]
This was due to a change that was made in how the frame buffer size was gotten. I have reverted that change back to older code that calls GLFW directly instead of asking rlgl for the info. This seems to fix the problem. It's worse on a high DPI display because the resize shifts the stencil by a lot.

If you update code you should be good now.

**Reply by dougreichard** [reply]
That worked thank you for the quick responses.
